src/App.jsx

```diff
@@ -1,5 +1,48 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import classNames from 'classnames';
+
+const clearButton = (setCurrentSelected) => (
+  <button 
+      data-cy="ClearButton" 
+      type="button" 
+      
+      onClick={() => 
+      {
+        setCurrentSelected('')
+      }}
+      className ="delete ml-3"
+      />
+)
```

Comment:
The `clearButton` function is being used as a component but is not capitalized. In React, component names should start with an uppercase letter. Consider renaming it to `ClearButton`.